dev-libs/libmelf/libmelf-0.4.0-r2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic toolchain-funcs
+
+DESCRIPTION="libmelf is a library interface for manipulating ELF object files"
+HOMEPAGE="http://www.hick.org/code/skape/libmelf/"
+SRC_URI="http://www.hick.org/code/skape/${PN}/${P}.tar.gz"
```

Comment:
While at it, could you update `HOMEPAGE` and `SRC_URI`, too?

```
dev-libs/libmelf
  HttpsUrlAvailable: version 0.4.0-r2: HOMEPAGE: HTTPS url available: http://www.hick.org/code/skape/libmelf/ -> https://www.hick.org/code/skape/libmelf/
  HttpsUrlAvailable: version 0.4.0-r2: SRC_URI: HTTPS url available: http://www.hick.org/code/skape/libmelf/libmelf-0.4.0.tar.gz -> https://www.hick.org/code/skape/libmelf/libmelf-0.4.0.tar.gz
```

Reply:
done, thanks